flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactory.java

```diff
@@ -322,6 +329,13 @@ private void validateConfigOptions(ReadableConfig config) {
 				SINK_MAX_RETRIES.key(),
 				config.get(SINK_MAX_RETRIES)));
 		}
+
+		if (config.get(MAX_RETRY_TIMEOUT).getSeconds() <= 0) {
+			throw new IllegalArgumentException(String.format(
+				"The value of '%s' option must be in second granularity and larger than 1 second, but is %s.",
```

Comment:
Maybe `larger than 0 second` instead of `larger than 1 second`?

Reply:
thanks @V1ncentzzZ ,I have changed to a clearer description